### DISCUSSION TITLE
Whether it can be receive RTP PS stream

### DISCUSSION THREAD

**Discussion by bighuangbee** [bluenviron/mediamtx#1266] [Questions]
hello，Whether it can be receive RTP PS stream，then output rtsp stream address？
If not, what can I do，thanks.

**Comment by aler9** [comment]
Hello, this is not currently planned since MPEG-PS streams are mostly used with DVDs and reliable medias and are not fit for network transfers. Nonetheless, ingesting UDP MPEG-TS streams is planned and will be added soon.

[a comment by an automated account is not shown]
